project/BuildDependencies/DownloadMingwBuildEnv.bat

```diff
@@ -28,15 +34,16 @@ IF NOT EXIST %MSYS_INSTALL_PATH% md %MSYS_INSTALL_PATH%
 IF NOT EXIST %MINGW_INSTALL_PATH% md %MINGW_INSTALL_PATH%
 IF NOT EXIST %TMP_PATH% md %TMP_PATH%
 
-cd scripts
+PUSHD %BUILD_DEPS_PATH%\scripts
 
 CALL get_msys_env.bat
 IF EXIST %TMP_PATH% rmdir %TMP_PATH% /S /Q
 CALL get_mingw_env.bat
 
-cd %CUR_PATH%
+POPD
 
 rem update fstab to install path
+IF NOT EXIST %MSYS_INSTALL_PATH%\etc md %MSYS_INSTALL_PATH%\etc
```

Comment:
This change does not appear to be necessary anymore. I'll revert it in the next commit.